packages/esm-ward-app/src/ward-workspace/ward-patient-notes/form/notes-form.component.tsx

```diff
@@ -29,14 +29,19 @@ const noteFormSchema = z.object({
 
 interface PatientNotesFormProps extends DefaultWorkspaceProps {
   patientUuid: PatientUuid;
+  emrConfiguration: EmrApiConfigurationResponse;
+  isLoadingEmrConfiguration: boolean;
+  errorFetchingEmrConfiguration: Error;
```

Comment:
Is there a reason you moved this from using to useEmrConfiguration to pass in as a prop?  I think SWR is smart enough to cache this request instead of making it multiple times, it seems clearer to just have useEmrConfiguration each time you need that resource. @chibongho?